### ISSUE TITLE
Says All Done But Is Still on Rebooting Screen

### ISSUE THREAD

**Issue by ghost** [checkra1n/BugTracker#216]
**What device + iOS version are you on?**
iPad Pro Generation 2. IOS 13.2

**What checkra1n version are you using?**
Version beta 0.9.2
**What are the steps to reproduce the issue?**
1. Launch the checkra1n
2. Do the steps to jailbreak
3. Upon finish, the checkra1n says all done, but device is still in rebooting screen.
...

**What do you expect, and what is happening instead?** 
I expected for the checkra1n launcher to be in my device.

**Any other info, error logs, screenshots, ...?**
![7E804D81-E8DA-47ED-AE50-EFA4773EA225](https://user-images.githubusercontent.com/57645013/68627906-d9436e00-0493-11ea-9e33-a14ed0073563.jpeg)

**Comment by ghost** [comment]
[REBOOTING AND RESTORING DEVICE DOESN'T WORK]

**Comment by GuyInHiding** [comment]
This is exactly what happens to me when I try to rejailbreak and I can't remove Cydia after restoring and resetting the device

**Comment by ghost** [comment]
> This is exactly what happens to me when I try to rejailbreak and I can't remove Cydia after restoring and resetting the device

Same thing, here. This didn’t happen to me before. It recently happened after I decided to rejailbreak.

**Comment by nullpixel** [comment]
Reboot holding the volume up button. Does it fix it?

**Comment by ghost** [comment]
![784648DA-5265-4905-8A8E-DD01D0907FAB](https://user-images.githubusercontent.com/57645013/68628660-11e44700-0496-11ea-81f4-cab68d7dd429.jpeg)
What do you mean by this? I tried and it just rebooted like normal. Let me try again. (like it’s just the normal reboot, not the one with the checkra1n icon one.

**Comment by mrmouseey** [comment]
Might try again.

**Comment by ghost** [comment]
> Reboot holding the volume up button. Does it fix it?

Oh hey! It works, thanks. Took a while for this to work.

**Comment by nullpixel** [comment]
awesome! Now uninstall the broken tweak and reboot, you should be good to go :)
